xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-test/xwiki-platform-appwithinminutes-test-pageobjects/src/main/java/org/xwiki/appwithinminutes/test/po/ClassEditorConstants.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.appwithinminutes.test.po;
+
+/**
+ * Stores the constants related to the class editor pages. 
+ *
+ * @version $Id$
+ * @since 13.2
+ * @since 12.10.6
+ */
+public interface ClassEditorConstants
+{
+    /**
+     * The message displayed when the canvas is empty.
+     */
+    String EMPTY_CANVAS_HINT = "Drag fields from the palette and drop them in this area.";
+}
```

Comment:
Not sure if it is the best way to do it but we need to store this constant in this module since it is still used in `xwiki-platform-distribution-flavor-test-ui` and would be duplicated otherwise. WDYT?